spanner/api/Spanner.Samples/CreateDatabaseAsyncPostgre.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright 2022 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_postgresql_create_database]
+
+using Google.Cloud.Spanner.Admin.Database.V1;
+using Google.Cloud.Spanner.Common.V1;
+using System;
+using System.Threading.Tasks;
+
+public class CreateDatabaseAsyncPostgreSample
+{
+    public async Task<Database> CreateDatabaseAsyncPostgre(string projectId, string instanceId, string databaseId)
+    {
+        DatabaseAdminClient databaseAdminClient = await new DatabaseAdminClientBuilder
+        {
+            // QuotaProject needs to be set for now while feature is in Pre-GA. May not be needed in final sample.
```

Comment:
I see it is, thanks! Since we won't merge this until the library has been released and the feature is GA, maybe remove the setting of QuotaProject. We'll run testst when we are ready to merge and if they fail we can add it back in. Just don't want to put out there something that's not needed.